tools/packaging/kata-deploy/helm-chart/kata-deploy/templates/runtimeclasses.yaml

```diff
@@ -21,7 +21,7 @@
   "qemu-snp" (dict "memory" "2048Mi" "cpu" "1.0")
   "qemu-tdx" (dict "memory" "2048Mi" "cpu" "1.0")
   "qemu-nvidia-gpu" (dict "memory" "4096Mi" "cpu" "1.0")
-  "qemu-nvidia-gpu-snp" (dict "memory" "4096Mi" "cpu" "1.0")
+  "qemu-nvidia-gpu-snp" (dict "memory" "16384Mi" "cpu" "1.0")
```

Comment:
Split all the runtime class changes out of this commit and  into a separate commit (same PR, different commit, and also bump the TDX one as we know it'll be needed).